src/components/authentication/SigninGithubWrapper.vue

```diff
@@ -0,0 +1,529 @@
+          <v-col cols="12" class="pa-0">
+            <div class="subtitle-2 label-text">
+              {{ $tc("caption.api_key", 1) }}
+            </div>
+            <div class="timer-box-wrapper">
+              <v-text-field
+                outlined
+                dense
+                v-model="apiKey"
+                required
+                :rules="rules.apiKey"
+              />
+            </div>
+          </v-col>
+          <v-col cols="12" class="pa-0">
+            <div class="subtitle-2 label-text">
+              {{ $tc("caption.jira_instance_url", 1) }}
+            </div>
+            <div class="timer-box-wrapper">
+              <v-text-field
+                outlined
+                dense
+                v-model="instanceUrl"
+                placeholder="mydomain.atlassian.net"
+                required
+                :rules="rules.instanceUrl"
+              />
+            </div>
+          </v-col>
+          <v-col cols="12" class="pa-0">
+            <v-btn
+              class="text-capitalize btn_signup"
+              color="primary"
+              fill
+              small
+              block
+              :loading="loading"
+              @click="signInWithApiKey()"
+            >
+              {{ $tc("caption.sign_in", 1) }}
+            </v-btn>
+          </v-col>
+        </v-row>
+      </v-form>
+    </div>
+    <v-snackbar v-model="snackBar.enabled" timeout="3000">
+      {{ snackBar.message }}
+      <template v-slot:action="{ attrs }">
+        <v-btn
+          color="primary"
+          text
+          v-bind="attrs"
+          @click="snackBar.enabled = false"
+        >
+          Close
+        </v-btn>
+      </template>
+    </v-snackbar>
+  </v-container>
+</template>
+<script>
+import axios from "axios";
+import dayjs from "dayjs";
+import uuidv4 from "uuid";
+import { Buffer } from "buffer";
+import { IPC_HANDLERS, IPC_FUNCTIONS } from "../../modules/constants";
+import yattIntegrationHelper from "../../integrations/YattIntegrationHelpers";
+import integrationHelper from "../../integrations/IntegrationHelpers";
+import githubIntegrationHelper from "../../integrations/GithubHelpers";
+export default {
+  name: "SigninGithubWrapper",
+  props: {
+    configItem: {
+      type: Object,
+      default: () => {},
+    },
+    credentialItems: {
+      type: Object,
+      default: () => {},
+    },
+    prevRoute: {
+      type: Object,
+      default: () => {},
+    },
+  },
+  watch: {
+    configItem: function (newValue) {
+      this.config = newValue;
+    },
+    credentialItems: function (newValue) {
+      this.credentials = newValue;
+    },
+    prevRoute: function (newValue) {
+      this.previousRoute = newValue;
+    },
+  },
+  data() {
+    return {
+      config: this.configItem,
+      credentials: this.credentialItems,
+      previousRoute: this.prevRoute,
+      connectWithCloudOAuth: true,
+      connectWithApi: false,
+      connectWithServerOAuth: false,
```

Comment:
This doesn't seem to be supported

Could you create a new issue for supporting GH server if we aren't implementing that now?

Reply:
Created https://github.com/yatt-ai/yattie/issues/112